src/cli/README.md

```diff
@@ -114,7 +114,7 @@ Done
 
 ### extpanid \<extpanid\>
 
-Set the Thread Extended PAN ID value.
```

Comment:
Agreed. Since OpenThread documentation started out by referring to "Thread", I am trying to see out those occurrences and convert them to "OpenThread". I will have to rely on you two and others to do that until I become more familiar with the details.